decidim/decidim
# Prevent notifications for blocked users

Prevent sending notifications to blocked users

#### :pushpin: Related Issues
- Fixes #13472 

#### Testing
Previous conditions for "User X"
  - User X has all notifications activated in realtime
  - User X follows the activity of the participation space “A”

Steps to follow as a site administrator:
  - Publish a content in space "A" and check if user X receive the notification. (He received. OK)
  - Block User X
  - Publish a content in space "A" and check if user X receive the notification. (He received. NOT OK)

Expected behavior:
  - Blocked users shouldn't receive any email notification (except block/unblock message)
